client/campaign_message_configuration/README.md

```diff
@@ -16,51 +16,26 @@ Linux or Windows
 King Phisher Minimum Required Version = 1.10.0 
 ```
 
-### Installing 
+### Use King Phisher Plugin Manager to install and enable plugin
 
 - [ ] Download plugin 

```

Comment:
this would need to be broken out to two sections.

one would be manually install
download
mv to config folder
enable in plugin manager

Through King Phisher itself
use the plugin manager to install and enable